clang/test/Driver/spirv-toolchain.cl

```diff
@@ -77,13 +77,3 @@
 
 // XTOR: {{llvm-spirv.*"}}
 // BACKEND-NOT: {{llvm-spirv.*"}}
-
-//-----------------------------------------------------------------------------
-// Check llvm-spirv-<LLVM_VERSION_MAJOR> is used if it is found in PATH.
-// RUN: mkdir -p %t/versioned
-// RUN: touch %t/versioned/llvm-spirv-%llvm-version-major \
-// RUN:   && chmod +x %t/versioned/llvm-spirv-%llvm-version-major
-// RUN: env "PATH=%t/versioned" %clang -### --target=spirv64 -x cl -c %s 2>&1 \
```

Comment:
Instead of creating a file, consider `%if !system-windows`

Reply:
Ok, updated the PR to use `%if !system-windows` instead - tested that it does seem to work as expected here.